docs/guide/website/sidebars.json

```diff
@@ -13,17 +13,6 @@
       "main/nlu",
       "main/emulator"
     ],
-    "Advanced Guides": [
```

Comment:
@Michael-N-M I have removed the pages that you renamed to place under deployment and infrastructures and I moved the rest at the bottom of the documentation. Please double check to make sure I didn't forget anything.